catalogs/catalog-jdbc-postgresql/src/main/java/com/datastrato/gravitino/catalog/postgresql/operation/PostgreSqlSchemaOperations.java

```diff
@@ -116,6 +115,19 @@ public String generateCreateDatabaseSql(
     return sqlBuilder.toString();
   }
 
+  /**
+   * Get the schema with the given name.
```

Comment:
could you add some description about the user should do an extra filter to get the exactly schema?

Reply:
Fixed